src/imitation/algorithms/preference_comparisons.py

```diff
@@ -299,6 +302,95 @@ def logger(self, value: imit_logger.HierarchicalLogger):
         self.algorithm.set_logger(self.logger)
 
 
+class MixtureOfTrajectoryGenerators(TrajectoryGenerator):
+    """A collection of trajectory generators merged together."""
+
+    members: Sequence[TrajectoryGenerator]
+
+    def __init__(
+        self,
+        members: Sequence[TrajectoryGenerator],
+        custom_logger: Optional[imit_logger.HierarchicalLogger] = None,
+        share_training_steps: bool = True,
```

Comment:
@AdamGleave do you think its more natural to have this default to True or False?

Reply:
Sorry for slow response, somehow missed this question. My inclination is to default to True but this is weakly held.